apps/nextjs-website/src/helpers/metadata.helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { translations } from '@/_contents/translations';
+import { ResolvedMetadata } from 'next';
+import { AbsoluteTemplateString } from 'next/dist/lib/metadata/types/metadata-types';
+import { Twitter } from 'next/dist/lib/metadata/types/twitter-types';
+
+export const getTitleFromMarkdown = (markdown: string): string => {
+  const lines = markdown.split('\n').filter((line) => line !== '');
+  const titles = lines.filter((line) => line.startsWith('#'));
+  const title = titles[0]?.replace('# ', '').trim() ?? '';
+  return title;
+};
+
```

Comment:
Take a look if this function is needed or if you can avoid to use it because the title of the pages is parsed by another function (in another comment there is an alternative solution to get the title, but you should check if it's ok for this use case).

If you think this method is still needed, then we should improve it (e.g. you have testes `should return the first title if the markdown has multiple titles` and it's ok, but IMHO you should also tests cases like `should return the first h1 title even it happens after an h2 subtitle` of case where pages do not have title. in this case the function is not working properly)